Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1064,31 +1070,31 @@ public extension Purchases {
 
 #if os(iOS) || targetEnvironment(macCatalyst) || VISION_OS
 
-    @available(iOS 16.4, *)
+    @available(iOS 16.0, *)
     @available(macOS, unavailable)
     @available(watchOS, unavailable)
     @available(tvOS, unavailable)
     func showStoreMessages() async {
-        await self.storeMessagesHelper.showStoreMessages(types: Set(StoreMessageType.allCases))
+        await self.storeMessagesHelper?.showStoreMessages(types: Set(StoreMessageType.allCases))
     }
 
-    @available(iOS 16.4, *)
+    @available(iOS 16.0, *)
     @available(macOS, unavailable)
     @available(watchOS, unavailable)
     @available(tvOS, unavailable)
     func showStoreMessages(forRawValues rawValues: Set<NSNumber>) async {
-        let storeMessageTypes = rawValues.map({ number in
-            StoreMessageType(rawValue: number.intValue) ?? StoreMessageType.billingIssue
-        })
-        await self.storeMessagesHelper.showStoreMessages(types: Set(storeMessageTypes))
+        let storeMessageTypes = rawValues.compactMap { number in
```

Comment:
@tonidero I changed this because I don't think we want to assume the type?

Reply:
I'm still confused why we need this raw value API since this isn't available in Obj-C, but we can address that in your PR.

Reply:
The idea behind this was to support Obj-C yeah, specially to support the hybrids still using Obj-C... Why do you say this isn't available in Obj-C? Afaik, the APIs should work there but maybe I'm missing something.

Reply:
> I'm still confused why we need this raw value API since this isn't available in Obj-C
We should still be able to make it available for Obj-C devs, right? Historically we've tried to make StoreKit's Swift-only APIs available in Obj-C through our own API